blazar/plugins/oshosts/host_plugin.py

```diff
@@ -636,9 +637,11 @@ def _matching_hosts(self, hypervisor_properties, resource_properties,
             ]:
                 allocated_host_ids.append(host['id'])
         if len(not_allocated_host_ids) >= int(min_host):
+            shuffle(not_allocated_host_ids)
```

Comment:
Does this break any tests? We should probably update the `test_physical_hosts_plugin.py` file with a test that the hosts are randomized.